src/main/java/com/sourcegraph/vcs/RepoUtil.java

```diff
@@ -50,11 +50,10 @@ public static RepoInfo getRepoInfo(@NotNull Project project, @NotNull VirtualFil
       remoteUrl = getRemoteRepoUrl(project, file);
       remoteUrl = doReplacements(codyProjectSettings, remoteUrl);
 
-      // If the current branch doesn't exist on the remote or if the remote
-      // for the current branch doesn't correspond with the sourcegraph remote,
+      // If the current branch doesn't exist on the remote
       // use the default branch for the project.
       remoteBranchName = getRemoteBranchName(project, file);
-      if (remoteBranchName == null || !remoteUrl.contains(remoteBranchName)) {
+      if (remoteBranchName == null) {
         remoteBranchName = codyProjectSettings.getDefaultBranchName();
       }
```

Comment:
@olafurpg, do you have any ideas what was it about?

Reply:
Don't recall any specific details here

Reply:
ok. anyway - it seems to work properly now